section-05-production-model-package/tox.ini

```diff
@@ -0,0 +1,75 @@
+# Tox is a generic virtualenv management and test command line tool. Its goal is to
+# standardize testing in Python. We will be using it extensively in this course.
+
+# Using Tox we can (on multiple operating systems):
+# + Eliminate PYTHONPATH challenges when running scripts/tests
+# + Eliminate virtualenv setup confusion
+# + Streamline steps such as model training, model publishing
+
+
+[tox]
+envlist = train,unit_tests,typechecks,stylechecks
+skipsdist = True
+
+[testenv]
+install_command = pip install {opts} {packages}
+deps =
+	-rrequirements/requirements.txt
+
+setenv =
+	PYTHONPATH=.
+
+[testemv:train]
+deps =
+	{[testenv]deps}
+
+# TODO: fix kaggle script
+commands=
+	python regression_model/train_pipeline.py
+
+
+[testenv:unit_tests]
+envdir = {toxworkdir}/unit_tests
+deps =
+	-rrequirements/test_requirements.txt
+
+setenv =
+	PYTHONPATH=.
+
+passenv =
+	KAGGLE_USERNAME
+	KAGGLE_KEY
+
+# TODO: fix kaggle script
+commands=
```

Comment:
I would strongly recommend add an article saying: please copy the files train.csv and test.csv that you downloaded in section 1 in the folder dataset. 

And avoid downlaoding the data from kaggle, it is extra-complication, for which I don't see the added value.